src/platforms/common/configuration/options.mdx

```diff
@@ -373,7 +373,7 @@ This is the maximum number of properties or entries that will be included in any
 <ConfigKey name="attach-screenshot" supported={["android", "apple.ios", "unity", "dotnet.xamarin"]}>
 
 Takes a screenshot of the application when an error happens and includes it as an attachment.
-Screenshots for hard crashes are not supported on iOS.
+Learn more in <PlatformLink to="/enriching-events/screenshots/">Enriching Events with Screenshots</PlatformLink>.
```

Comment:
Ack! In my rush to get this reviewed for you, I missed two things. The word "with" should be upper case. But also, this isn't the name of any actual page in our docs, so I'm leaving two suggestions here. Pick the one you like most. 

```suggestion
Learn more in our <PlatformLink to="/enriching-events/screenshots/"> Screenshots</PlatformLink> documentation.
```

Reply:
```suggestion
Learn more in <PlatformLink to="/enriching-events/screenshots/">Screenshots</PlatformLink>.
```